src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -10,20 +10,91 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 
+<!--
+
+NOTE: This file is imported from the following contexts, so be aware when writing Conditions and thinking about evaluation order:
+* cross-targeting (multi-TFM) builds.
+  * these may not have a RID or RIDs set, but still need to be able to pack tools
+  * IsCrossTargetingBuild is set to true in this case
```

Comment:
Thank you for this comment, it's helpful, but phew this is complicated (not your fault)